### ISSUE TITLE
Data not showing with Dekstop sample

### ISSUE THREAD

**Issue by nobodykid** [Azure/connectthedots#249]
I'm trying to use the Desktop sample in /Devices/DirectlyConnectedDevices/NodeJS /Desktop

When I ran the code, it clearly sent the message

![console](https://cloud.githubusercontent.com/assets/12664445/18252563/50a76baa-73b9-11e6-8086-c1d50d40571a.JPG)

but the web didn't show anything. And also, I got similar message like #173 

![web](https://cloud.githubusercontent.com/assets/12664445/18252603/7e4e8f66-73b9-11e6-8b55-a495e8a54c30.JPG)

Did something wrong happened, or is there a step in deploying process that I missed?

**Comment by olivierbloch** [comment]
Hi @nobodykid 
THe error message at the bottom of the site seems to be indicating that the Websocket connection between the site frontend and backend failed. Make sure you have Websockets enabled in your Website settings in the Azure portal.
Also, you can verify that the data is indeed sent to IoT Hub using the IoTHub-explorer tool as described in [this doc](https://github.com/Azure/azure-iot-sdks/blob/master/doc/manage_iot_hub.md)

**Comment by nobodykid** [comment]
I have enabled the setting based from instruction in Website Publish, and also the connection between IoT Hub and device seems working fine (the message counter in IoT Hub changed)

It seems the problem lies in the website. Because I got the same bug in #246 I had to manually publish the web through Visual Studio. During deployment, I didn't change any settings (the WebSockets option is already ON in Azure App Service). Could there be any steps/settings that I probably missed?
